.github/workflows/release.yml

```diff
@@ -225,16 +225,16 @@ jobs:
         with:
           pattern: '*-compact'
 
-      - name: Create compact-wasms.zip
+      - name: Create logging enabled runtimes
         run: |
-          zip -j compact-wasms.zip */*.compact.compressed.wasm
+          zip -j polkadot-fellows_logging_runtimes-v${{ env.SPEC }}-wasms.zip */*.compact.compressed.wasm
```

Comment:
not sure if you can use `env.SPEC`, which is related to the one runtime, maybe we patch just one runtime and also this is not matrix-ed step